portfolio.css

```diff
@@ -211,3 +207,731 @@ body {
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+}
+
+.indicator_horizontal {
+  width: 48px;
+  height: 4px;
+  border-radius: 24px;
+}
+
+.main-content-cards {
+  display: grid;
+  grid-row: 1 / 7;
+  row-gap: 24px;
+  padding-top: 62px;
+  align-items: center;
+  padding-left: 64px;
+}
+
+.card-work-1 {
+  display: flex;
+  flex-direction: column;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.image {
+  width: 359px;
+  height: 252px;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  padding: 16px;
+  width: 327px;
+  height: 222px;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.header-work-1 {
+  width: 295px;
+  height: 57px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: #3a4a42;
+  margin: 0;
+  padding-top: 16px;
+}
+
+.list-languages {
+  width: 327px;
+  height: 32px;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  padding: 0;
+  align-items: center;
+  margin: 0;
+}
+
+.ruby {
+  width: 120px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.css {
+  width: 45px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.javascript {
+  width: 82px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.html {
+  width: 51px;
+  padding: 8px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  list-style: none;
+}
+
+.button-frame {
+  width: 305px;
+  height: 69px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.btn-green {
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  margin: 0;
+}
+
+.btn-green:hover {
+  border: 1px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
```

Comment:
- [ ] As mentioned by the previous reviewer. Please make sure that your button hover and pressed state matches the one in the figma design template. You can use the `:active` pseudo-class for the pressed state
![Screenshot 2022-08-20 203951](https://user-images.githubusercontent.com/94551484/185763627-ce6a6ba5-cc72-401b-8d1c-2d803bbcbd54.png)
.